lib/util/mutil.ml

```diff
@@ -506,13 +507,23 @@ let input_lexicon lang ht open_fname =
              && String.unsafe_get line 2 = ':'
              && String.unsafe_get line 3 = ' '
         then
+        begin
           let k2 = String.sub line 4 (String.length line - 4) in
-          Opt.iter (Hashtbl.replace ht k) (Hashtbl.find_opt ht k2) ;
+          Hashtbl.replace tmp k k2;
           key ()
+        end
         else trad k
-      | None -> key ()
+      | None -> key () ;
   in
-  key ()
+  key () ;
+  if Hashtbl.length tmp > 0 then
+  Hashtbl.iter (fun k k2 -> 
+      match Hashtbl.find_opt ht k2 with
+      | Some entry -> Opt.iter (Hashtbl.replace ht k) (Some entry) 
```

Comment:
```diff
-     | Some entry -> Opt.iter (Hashtbl.replace ht k) (Some entry)
+     | Some entry -> Hashtbl.replace ht k entry
```

Reply:
yep.
Should the warning be sent via syslog? 
When I tried, it said module Gwd_lib.GwdLog undefined!!

Reply:
You can extract the syslog part from this PR:

https://github.com/geneweb/geneweb/pull/1227/commits/a519ff79fbcd975dc0ddce1b08f06151f0b6ffb6

Reply:
Can't! I must be missing something!!

Reply:
I'll pick it for you.

Reply:
Here we are: https://github.com/geneweb/geneweb/commit/c2a4ee2565fb17e90f5d7ac3a248a60e888483a5